npm/podController.go

```diff
@@ -489,16 +517,15 @@ func (c *podController) syncAddAndUpdatePod(newPodObj *corev1.Pod) error {
 		if err = ipsMgr.DeleteFromSet(podIPSetName, cachedNpmPodObj.PodIP, podKey); err != nil {
 			return fmt.Errorf("[syncAddAndUpdatePod] Error: failed to delete pod from label ipset with err: %v", err)
 		}
-	}
-
-	// Add the pod to its label's ipset.
-	for _, addIPSetName := range addToIPSets {
-		log.Logf("Adding pod %s to ipset %s", newPodObj.Status.PodIP, addIPSetName)
-		if err = ipsMgr.AddToSet(addIPSetName, newPodObj.Status.PodIP, util.IpsetNetHashFlag, podKey); err != nil {
-			return fmt.Errorf("[syncAddAndUpdatePod] Error: failed to add pod to label ipset with err: %v", err)
+		// {IMPORTANT} The order of compared list will be key and then key+val. NPM should only append after both key
```

Comment:
(Same as namespace comment) Can we resolve this issue in this PR? As I understood this issue and we discussed this before, but if possible, it is better to resolve this issue to avoid future possible issue.
Current GetLabelKVFromSet returns key and value. So better to returns (string, string) in the GetLabelKVFromSet
function and use "removedKey" and "removedValue" in this function